ecosystem/platform/server/config/tailwind.config.js

```diff
@@ -23,6 +23,18 @@ module.exports = {
           '400': '#1bd7a4',
           '500': '#187e65',
         },
+        surf: {
```

Comment:
Let's delete the changes to `teal` in this file now that we have the full palette.